tests/ops/test_qubit_ops.py

```diff
@@ -2820,8 +2820,13 @@ def test_qubit_unitary_jax(self, U, num_wires):
             (qml.RZ(0.3, wires=0).matrix, qml.RZ, [0.3]),
             (qml.RZ(-0.5, wires=0).matrix, qml.RZ, [-0.5]),
             # Next set of gates are non-diagonal and decomposed as Rots
+            (
+                np.array([[0, -0.98310193 + 0.18305901j], [0.98310193 + 0.18305901j, 0]]),
+                qml.Rot,
+                [0, -np.pi, -5.914991017809059],
+            ),
             (H, qml.Rot, [np.pi, np.pi / 2, 0]),
-            (X, qml.Rot, [0.0, np.pi, np.pi]),
+            (X, qml.Rot, [0.0, -np.pi, -np.pi]),
```

Comment:
Was this because the matrix equivalences were not being checked?

Reply:
Nope this is just because now for the off-diagonal-only case, theta is by default set to `-np.pi`.